### PR TITLE
write apple walkthrough

### PR DESCRIPTION
**Pull request by skotchpine**
This should help with solving a few outstanding issues (#18, #33, #22)

### CODE REVIEW

**Comment by skotchpine** [comment]
Addressing issues: #18 #33

**Review by Gedrovits — DISMISSED** [review]
That is *very* useful and informative addition to base README.

**Comment by nicbet** [comment]
Would be great to get this merged - exactly the information I spent quite a bit of time looking for.

**Review by martino-dot — APPROVED** [review]

**Comment by ShawnAukstak** [comment]
After struggling to figure this out myself, I came across this PR.  I just used it and saved me a lot of time.

Hope it can get merged in 🙏

File: README.md [hunk 1]
```diff
@@ -34,6 +34,77 @@ Rails.application.config.middleware.use OmniAuth::Builder do
...
+    - make sure the Primary App ID matches the App ID you configured earlier
+    - save the "Sign In with Apple" capability
+    - click "continue" to finish the Key config (you will be prompted to _**Download Your Key**_)
+    - Apple will give you a `.p8` file, keep it safe and secure (don't commit it).
+
+### Mapping Apple Values to OmniAuth Values
+  - your `:team_id` is in the top-right of your App Id config (aka _**App ID Prefix**_), it looks like: `H000000B`
+  - your `:client_id` is in the top-right of your Services Id config (aka _**Identifier**_), it looks like: `com.example`
+  - your `:key_id` is on the left side of your Key Details page, it looks like: `XYZ000000`
+  - your `:pem` is the content of the `.p8` file you got from Apple, _**with an extra newline at the end**_
```
Comment (joshdchap) [inline README.md:95]: is `:pem` not the private key that you generate from the `.p8`?

File: README.md [hunk 2]
```diff
@@ -34,6 +34,77 @@ Rails.application.config.middleware.use OmniAuth::Builder do
 end
 ```
 
+## Configuring "Sign In with Apple"
+
+_other Sign In with Apple guides:_
+- ["How To" by janak amarasena (2019)](https://medium.com/identity-beyond-borders/how-to-configure-sign-in-with-apple-77c61e336003)
+- [the docs, by Apple](https://developer.apple.com/sign-in-with-apple/)
```
Comment (fidalgo) [inline README.md:41]: I would add the help docs https://help.apple.com/developer-account/#/dev1c0e25352
It helps in case the reference get's out-of-date.

**Review by nov — APPROVED** [review]

**Pull request merged**